core/storage/config/gae_models.py

```diff
@@ -33,63 +33,6 @@
 datastore_services = models.Registry.import_datastore_services()
 
 
-class ConfigPropertySnapshotMetadataModel(
```

Comment:
Will there be or was there a Beam job for removing these from the datastore?

Reply:
Ohh, Umm, We do not have that right now, I was not aware if we were planning to do that, is the datastore cleanup required?

Reply:
It'd be cleaner to clean it up, but I think since we are deprecating this model entirely and not reading/writing from it, we can probably leave it as-is -- it doesn't really cause any harm IMO.

@vojtechjelinek do you see anything that could go wrong if we proceed this way? I feel like at some point in the future we could just delete all the stuff that doesn't have a corresponding *Model class in the storage layer in the codebase.

Reply:
Yeah, I mainly commented so that we do not forget about it. I think it can be removed manually through the Google Cloud interface too.